build.sbt

```diff
@@ -1,5 +1,18 @@
 sbtPlugin := true
 
+enablePlugins(SbtPlugin)
+
+scriptedBufferLog := false
+
+scriptedLaunchOpts := {
+  scriptedLaunchOpts.value ++
+    Seq("-Xmx1024M", "-Dplugin.version=" + version.value)
+}
+
+Test / test ~= { _: Unit =>
```

Comment:
Hmm, `~=` will keep and be running the existing tests, after they complete it will run the `scripted` task, right? I missed that and replaced the `test` task in my commit.

Reply:
It doesn't matter for this project because there was no unit test yet...

Reply:
But anyway, running existing tests (if some) is always good.